docs/cheatsheet/cheat_sheet_gen.clj

```diff
@@ -127,35 +133,40 @@ Version & \\cmd{processing-version} \\\\
 (defn cheat-sheet-dynamic
```

Comment:
In `cheat-sheet-docs` function `processing-version` no longer available so it can be removed.